packages/ripple-address-codec/src/xrp-codec.ts

```diff
@@ -2,19 +2,19 @@
  * Codec class
  */
 
+import { sha256 } from '@xrplf/isomorphic/sha256'
 import baseCodec = require('base-x')
 import type { BaseConverter } from 'base-x'
-import createHash = require('create-hash')
 
-import { seqEqual, concatArgs } from './utils'
+import { seqEqual, concatArgs, Sequence } from './utils'
 
 class Codec {
-  private readonly _sha256: (bytes: Uint8Array) => Buffer
+  private readonly _sha256: (bytes: Sequence) => Uint8Array
```

Comment:
Why are all these changes needed? Is it basically the list of supported types expanding?

Reply:
I wrote this part but it was months ago now.

But yeah, it's basically allowing hashing of number[], Uint8Array iirc

And I think part of it was the removal of Buffer to reduce bundling size/dependencies

Reply:
Like sublimator said this is part of removing `Buffer`.  I believe this removes Buffer from the address-codec the rest will come in another PR.

Reply:
Shouldn't the removal of Buffer be in a separate PR then?

Reply:
It will be. Most of the spots in this PR was done to make moving to noble libs easier and is focused on the address codec package.